src/fsharp/FSharp.Compiler.Service.netcore/output/FSIstrings.fs

```diff
@@ -0,0 +1,329 @@
+namespace FSIstrings
```

Comment:
Of course, that's there just temporary until I get the fssrgen working (the bootstrap fssrgen does not generate netcore-compatible code right now).